NtupleTools/python/templates/electrons.py

```diff
@@ -33,6 +33,7 @@
     objectMVATrigCategory = '{object}.userFloat("BDTIDTrigCategory")',
     objectMVATrigWP80 = '{object}.userFloat("MVATrigWP80")',
     objectMVATrigWP90 = '{object}.userFloat("MVATrigWP90")',
+    objectWWLoose = '{object}.userFloat("WWLoose")',
```

Comment:
Could you please add a check for the userFloat's existence? Something like `'? {object}.hasUserFloat("WWLoose") ? {object}.userFloat("WWLoose") : -999'`. We need to do this for all of the userFloats in our templates, really.